apps/aecore/lib/aecore/oracle/tx/oracle_extend_tx.ex

```diff
@@ -66,6 +66,7 @@ defmodule Aecore.Oracle.Tx.OracleExtendTx do
         data_tx
       ) do
     sender = DataTx.main_sender(data_tx)
+    # ? {:ok, identified_oracle} = Identifier.create_identity(sender, :oracle)
```

Comment:
is this not needed?

Reply:
Yes, its not needed anymore and will be removed.